indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -444,6 +444,19 @@ $dropdown-transition: $dropdown-transition-step ease-out;
                 width: 10em;
             }
 
+            > .form-row.i-has-action.i-synced-input {
```

Comment:
No, seriously. Why so many `i-whatever` within `i-form`? I think just having the `i-` on the base class is more than enough. It looks **very** funny like this.